data/systemd-env-generators/50-snapd.sh.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+#
+# Ensure /snap/bin is in the PATH
+
+echo PATH=$PATH:@SNAP_MOUNT_DIR@/bin
```

Comment:
Nice, I assume this supersedes other environment setting mechanism for adding /snap/bin to PATH.

Btw, http://in.waw.pl/~zbyszek/blog/environmentd.html via discussion on https://new.zygoon.pl/post/case-study-snapd-on-centos/ :-)